src/legacy/core_plugins/kibana/public/visualize/wizard/step_2.html

```diff
@@ -7,17 +7,19 @@
       data-transclude-slot="topLeftCorner"
       use-links="true"
       omit-current-page="true"
-      page-title="'Choose search source'"
+      page-title="translations.pageTitle"
```

Comment:
Is there a specific reason we're not just calling the i18n filter here inline on the the key, but attach it to the controller? I would personally prefer more if we would just inline this here, since it doesn't seem to be used somewhere else.